inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -71,24 +71,26 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
 }
 
 
-@mixin bslib-box-shadows {
+@mixin bslib-dashboard-cards {
 
-  // Add box-shadow & remove border from 'top-level' card()/layout_sidebar()
-  .bslib-card, .bslib-sidebar-layout {
-    // Shadows on top-level cards/sidebars, not in nested components
-    box-shadow: $box-shadow;
+  @if ($bslib-enable-shadows) {
```

Comment:
It feels like we should get rid of this, especially since it's effect is scoped to a dashboard page context (i.e., if `$bslib-enable-dashboard` is `false` and `$bslib-enable-shadows` is `true`, `$bslib-enable-shadows` won't actually do anything)